test/test_transformers.py

```diff
@@ -2111,6 +2111,36 @@ def test_fused_kernels_nested_broadcasting_error_cases(self):
 
 
 
+    def test_padding_and_src_mask_bool(self):
```

Comment:
can you move this test to the TestTransformers class

Reply:
done